commercetools-sdk-base/src/main/java/io/sphere/sdk/annotations/HasNoUpdateAction.java

```diff
@@ -0,0 +1,14 @@
+package io.sphere.sdk.annotations;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+@Retention(RetentionPolicy.SOURCE)
+@Target({ElementType.METHOD})
+/**
+ * This annotation marks methods that will be ignored while generating update actions with {@link HasUpdateActions}
```

Comment:
I like your idea to have this annotation. And great that you added javadoc too 👍